### PR TITLE
Issue 46 lazy import

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by doutriaux1** [comment]
@sterlingbaldwin @downiec @William-Hill in the end we need lazy_import because lazy_object_proxy does not work on modules.

**Comment by doutriaux1** [comment]
Still it cleans the build and README

File: README.md
Comment (downiec) [inline README.md]: # Install azy_import -> # Install lazy_import
Minor comment typo there

**Review by downiec — APPROVED** [review]

File: README.md
Comment (doutriaux1) [reply README.md]: fixed

Comment (downiec) [reply README.md]: Looks good! Should we wait for Sterling or William to review before merging?

**Pull request merged**
